tdrs-backend/tdpservice/test/test_admin.py

```diff
@@ -0,0 +1,47 @@
+"""System wide tests.
+
+Fixtures are used to create a regular user
+and an admin user. The regular user has is_staff and is_superuser
+set to false, while the admin_user has both set to true.
+"""
+
+import pytest
+from django.urls import reverse
+from rest_framework import status
+
+
+@pytest.mark.django_db
+def test_can_access_admin(client, admin_user):
+    """Test an authenticated admin_user can access the admin console."""
+    client.login(username=admin_user.username, password='test_password')
+    url = reverse('admin:index')
+    response = client.get(url)
+    assert response.status_code == status.HTTP_200_OK
+    assert "Django administration" in response.rendered_content
+    assert "Log entries" in response.rendered_content
+    assert "Users" in response.rendered_content
```

Comment:
Nit: you could enhance these assertions by iterating the admin site registry. This may make sense as a different test but it would confirm that all registered models are available to an admin user:
```
from django.utils.text import capfirst
for model in admin.site._registry.keys():
    assert capfirst(model._meta.verbose_name_plural) in response.rendered_content
```
This is actually the same way that the admin site builds the section headers internally so if access is granted as expected it should be a perfect match. [Reference](https://docs.djangoproject.com/en/1.8/_modules/django/contrib/admin/sites/)

Reply:
I like this idea - just implemented a slight variant and added some more tests for staff user as well